.vscode/settings.json

```diff
@@ -15,5 +15,25 @@
     "devel",
     "freedesktop",
     "nofilesystem"
-  ]
-}
+  ],
+  "json.schemas": [
```

Comment:
This is not correct, it should be defined in package.json instead otherwise it won't work

Reply:
```json

    "jsonValidation": [
      {
        "fileMatch": [
          "/*.*.*.json",
          "/*.*.*.*.json",
          "/*.*.*.*.*.json"
        ],
        "url": "./myschema.json"
      }
    ]
```
something like this works for the json, no idea how to setup the yaml one though

Reply:
That‘s why it‘s a draft.
I commited it so that it is easier for you to test 😉

Reply:
Sure, I just mentioned it because i didn't notice it was using the settings.json file. It works fine btw ^^

Reply:
Glad to hear that